specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -232,15 +232,21 @@
             "$ref": "#/parameters/DiskPoolName"
           },
           {
-            "$ref": "#/parameters/DiskPool"
+            "$ref": "#/parameters/DiskPoolUpdate"
```

Comment:
Suggest to define 3 objects for：
1. Create DiskPool: DiskPoolCreateParameter
2. Update DiskPool: DiskPoolUpdateParameter
3. Server returned DiskPool: DiskPool

This can make the swagger be flexiable for the create/update API change, like some properties will be return from server, but won't be set in create/update. 
And we can also do this from IscsiTargets create/update.

BTW, we use the similar 3 parameters in create/update storage account, and it has good experience when we add more properties to the 2 APIs.